Tests/scripts/update_conf_json.py

```diff
@@ -127,9 +128,16 @@ def main():
     existing_test_playbooks = load_test_data_from_conf_json()
     with ProcessPool(max_workers=os.cpu_count(), max_tasks=100) as pool:
         for pack_name in os.listdir(PACKS_DIR):
+            logging.debug(f'Collecting pack: {pack_name} tests to add to conf.json')
             future_object = pool.schedule(generate_pack_tests_configuration,
-                                          args=(pack_name, existing_test_playbooks), timeout=20)
-            future_object.add_done_callback(update_new_conf_json)
+                                          args=(pack_name, existing_test_playbooks), timeout=30)
+            try:
+                future_object.add_done_callback(update_new_conf_json)
+                logging.debug(f'Successfully added pack: {pack_name} test  to conf.json')
+
+            except Exception as err:
+                logging.error(f'Failed adding pack: {pack_name} tests:\n{err}')
+                sys.exit(1)
```

Comment:
Catching the exception here will not help you, you should add it to the `update_new_conf_json` method instead

Reply:
If what you're trying to achieve is to fail the build - add `raise` to line 124